### ISSUE TITLE
Request: original search result files in KNIME

### ISSUE THREAD

**Issue by julianu** [OpenMS/OpenMS#3161]
If possible, it would be nice to have an option for the search engine adapters in KNIME to export the original search results. Like MSGF does this already with the mzId file, it would be nice to also have the Mascot DAT, X!Tandem XML files and so on.

**Comment by algom** [comment]
I also agree with this

**Comment by hendrikweisser** [comment]
Not really a solution, but perhaps helpful to know: If you run the search engine adapter at a higher debug level (e.g. "debug 10"), it should keep the original search result (possibly together with other files) in a temporary directory - which would normally get deleted. It should write a log message: "Keeping temporary files in directory ...", to tell you where to look.

**Comment by julianu** [comment]
Yep, @hendrikweisser, that works for getting the file outside the workflow. But not really to process it further in a workflow, as you already stated.

**Comment by jpfeuffer** [comment]
bump!

**Comment by julianu** [comment]
Sure, stll missing the "native" output are:
- CompNovo and CompNovoCID
- CruxAdapter
- MascotAdapterOnline (here dat-files would be great)
- MyriMatchAdapter
- OMSSAAdapter

The CometAdapter has the output, but the output port information is a bit misleading: it states "Output file - for Percolator input [tsv]", though it basically is the comet TSV output.
